### PR TITLE
consolidate airflow DAGs documentations and fix image paths

### PR DESCRIPTION
**Pull request by Lee-W**
closes: #111

### CODE REVIEW

File: docs/airflow/README.md [hunk 1]
```diff
@@ -1,5 +1,209 @@
+<br />
+
+<p align="center">
+  <img src=".../_static/images/ingestion.png" />
```
Comment (sunank200) [inline docs/airflow/README.md:4]: Is this path still correct?

File: docs/airflow/README.md [hunk 2]
```diff
@@ -1,5 +1,209 @@
...
+astro dev bash -w
+```
+
+- Start Streamlit
+```bash
+cd include/streamlit
+python -m streamlit run ./streamlit_app.py
+```
 
-- [astro-cli](https://docs.astronomer.io/astro/cli/overview)
```
Comment (sunank200) [inline docs/airflow/README.md:5]: This is required as a pre-requisite to run this. It should be there

**Review by sunank200 — CHANGES_REQUESTED** [review]
I think the images are not visible correctly in airflow docs

File: docs/airflow/README.md (hunk 1 above)
Comment (Lee-W) [reply docs/airflow/README.md]: Nope. Thanks for reminding me! Just fixed

File: docs/airflow/README.md (hunk 2 above)
Comment (Lee-W) [reply docs/airflow/README.md:5]: Thanks! Just added it back.

**Review by sunank200 — APPROVED** [review]

**Pull request merged**